test/Token.test.js

```diff
@@ -1,34 +1,64 @@
 const { expectRevert, BN, expectEvent, time } = require('openzeppelin-test-helpers');
 
 const Token = artifacts.require('Token');
+const Contribution = artifacts.require('Contribution');
 
 contract('Token', function (accounts) {
   const owner = accounts[0];
   const recipient = accounts[1];
-  const amount = new BN(100);
+  const tokenSupply = new BN(50);
+  const rate = 10**18;
+  const amountEth = web3.utils.toWei("1", "ether");
+  const amountTokens = amountEth / rate;
   const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
 
   beforeEach(async function () {
-    token = await Token.new(1533114000, 1596272400, 1000);
+    token = await Token.new(1533114000, 1596272400, tokenSupply);
     // Wednesday, August 1, 2018 9:00:00 AM
     // Saturday, August 1, 2020 9:00:00 AM
     tokenAddress = token.address;
     tokenInstance = await Token.at(tokenAddress);
+
+    contribution = await Contribution.new(tokenAddress);
+    contributionAddress = contribution.address;
+    contributionInstance = await Contribution.at(contributionAddress);
+
+    // owner gives approval to contributionContract to transfer on their behalf
+    await token.approve(contributionAddress, tokenSupply);
   });
 
   describe('transfer', function () {
-    it('emits a Transfer event on successful Transfers', async function () {
-      const { logs } = await token.transfer(recipient, amount, { from: owner });
-      await expectEvent.inLogs(logs, 'Transfer', {to: recipient, value: amount });
+    it('emits a Sent event on successful Transfers', async function () {
```

Comment:
you are already testing for this in Contribution.test.js so you can remove this.